pyproject.toml

```diff
@@ -38,6 +38,7 @@ psycopg2 = { version="^2.9.6", optional=true }
 databricks-sql-connector = { version="^2.9.3", optional=true }
 elasticsearch = { version="^7.17.3", optional=true }
 pymysql = { version="^1.1.0", optional=true, extras=["rsa"] }
+sqlalchemy-redshift = "^0.8.14"
```

Comment:
@WeryZebra-Yue 
`sqlalchemy-redshift` has to be optional. `sqlalchemy-redshift = {version="^0.8.14", optional=true}`